changelog.md

```diff
@@ -2,6 +2,30 @@
 
 ## 1.1.0~rc1
 
+### Web Applications
+
+* Reordered and updated wording of manage.py - #4850, #4858
+* Remove python 2 support in securedrop server code - #4859
+
+### Operations
+
+* Improve specification of securedrop-app-code dependencies - #4876
+* Added check for admin password in GUI updater - #4877
+* Updated dependencies for app server and admin workstation - #4865, 4884
+* Fixes to packaging #4870, #4871
```

Comment:
Nit: formatting of the changelog lines is a bit different than the rest of the changelog, i.e. in the past it's been:

`* Description of item (#NNNN).`

Reply:
I mistakenly copied the formatting of earlier releases, now fixed